src/pages/App.js

```diff
@@ -8,10 +8,13 @@ import {
   SideMenu,
   AboutMe
 } from './styled'
+import Woman from '../assets/Woman.jpg'
```

Comment:
```suggestion
import Woman from 'assets/Woman.jpg'
```
No `jsconfig.json` tem uma configuração para utilizar o caminho absoluto a partir da pasta `src`, sendo assim tudo que estiver dentro de `src` você pode acessar diretamente, quando estiver no mesmo nível pode usar `./path` mas quando precisar andar mais de um nível ai melhor fazer como na sugestão.

Reply:
Muito bom saber disso, Cristiano! Fiz a modificação! Não conhecia esse recurso, facilita o trabalho de acessar as pastas.